sugar-slug/Utils/RigidBodyUtils.cs

```diff
@@ -0,0 +1,71 @@
+﻿using sugar_slug.Engine.Physics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace sugar_slug.Utils {
+    class RigidBodyUtils {
+        public RigidBodyUtils() {
+
+        }
+        public static Vector GetCenter(RigidBody body) {
+            return (body.Size / 2);
+        }
+        public static Vector GetTopLeft(RigidBody body) {
+            return (body.Position);
+        }
+        public static Vector GetTop(RigidBody body) => 
+            new Vector(
+                body.Position.X + body.Size.Y / 2, 
+                body.Position.Y, 
+                body.Position.Z
+        );
+
+        public static Vector GetTopRight(RigidBody body) =>
+            new Vector(body.Position.X + body.Size.X, 
+                body.Position.Y, 
+                body.Position.Z
+        );
+
+        public static Vector GetMiddleLeft(RigidBody body) => 
+            new Vector(body.Position.X, 
+                body.Position.Y + (body.Size.Y / 2), 
+                body.Position.Z
+        );
+
+        public static Vector GetMiddle(RigidBody body) => 
+            new Vector(body.Position.X + (body.Size.X / 2),
+                body.Position.Y + (body.Size.Y / 2), 
+                body.Position.Z
+        );
+
+        public static Vector GetMiddleRight(RigidBody body) => 
```

Comment:
You could just name it "Right"

```suggestion
        public static Vector GetRight(RigidBody body) => 
```